jobs/scheduler/spec

```diff
@@ -5,7 +5,6 @@ templates:
   bpm.yml.erb: config/bpm.yml
   liquibase.properties: bin/liquibase.properties
   scheduler.yml.erb: config/scheduler.yml
-  install_crt_keystore.sh: bin/install_crt_keystore
   install_crt_truststore.sh: bin/install_crt_truststore
```

Comment:
`install_crt_truststore.sh` cannot be removed as it is still used for the MySQL use case. As we know from https://github.com/cloudfoundry/app-autoscaler-release/discussions/2900 that people are still using MySQL but we do not have the CI or infra to test it, I'll leave it in for now.